wordpress-mobile/WordPress-iOS
# Update pod stats version.

Updates the Stats pod to a version that doesn't use sub specs.

To test:
 - Launch the app
 - Open the stats of one site
 - Navigate trough it and check if openURL and network activity indicator are working correctly.
 - Activate the today extension

Needs review: @nheagy check if this work correctly with the new cocoapods and @astralbodies see if stats are working correctly.
